src/vr_update_mps.cpp

```diff
@@ -29,7 +29,7 @@ bool vr_update_mps::crcCheckSum(){
      }
 
     ret = i2c_smbus_read_i2c_block_data(fd, CRC_ADDR, BYTE_COUNT_4, rdata);
-    if (rdata < SUCCESS)
+    if (ret < SUCCESS)
```

Comment:
is this bug? same for others also , if it is bug use diffrent commit to fix this.

Reply:
These are warnings which are not reported as part of C++17.
But when migrated to C++23 , this is throwing error